pysces/__init__.py

```diff
@@ -459,7 +459,23 @@ def enablePandas(var=True):
 from .PyscesModel import PysMod as model
 from .PyscesModel import ScanDataObj as ScanDataObj
 
-PyscesModel.interface = interface
+def loadSBML(sbmlfile, sbmldir=None, pscfile=None, pscdir=None):
```

Comment:
I like these functions, however, as historically the functions they call were meant as an import, they don't have any error checking or validation that the conversion took place correctly. In future convertSBML2PSC can probably become more intelligent, something for the future @bgoli